Tasks/Calculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿namespace Tasks
+{
+    /// <summary>
+    /// Calculator
+    /// </summary>
+    public class Calculator
+    {
+        /// <summary>
+        /// Calculate operations
+        /// </summary>
+        /// <param name="a">
+        /// First number
+        /// </param>
+        /// <param name="b">
+        /// Second number
+        /// </param>
+        /// <param name="actions">
+        /// Math operators
+        /// </param>
+        /// <exception cref="DivideByZeroException">
+        /// Divide by zero exception
+        /// </exception>
+        /// <exception cref="Exception">
+        /// Exception of zero numbers of roots
+        /// </exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        /// Argument out of range exception
+        /// </exception>
```

Comment:
We don't write exception in summary. But we write type that will be returned. Fix it please